src/accelerate/test_utils/testing.py

```diff
@@ -63,7 +63,7 @@ def get_backend():
     elif is_cuda_available():
         return "cuda", torch.cuda.device_count(), torch.cuda.memory_allocated
     elif is_mps_available():
-        return "mps", 1, torch.mps.current_allocated_memory()
+        return "mps", 1, (torch.mps.current_allocated_memory() if hasattr(torch, "mps") else 0)
```

Comment:
Rather than doing this, can we add a new arg in `is_mps_version` for `min_version` and check instead? I'd much rather do that than a `hasattr`

You can see an equivalent one for